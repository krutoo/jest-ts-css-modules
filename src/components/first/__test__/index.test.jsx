import React from 'react';
import { mount } from 'enzyme';
import { First } from '..';

describe('<First />', () => {
  it('should match snapshot', () => {
    const wrapper = mount(
      <First />
    );

    expect(wrapper).toMatchSnapshot();
  });
});
