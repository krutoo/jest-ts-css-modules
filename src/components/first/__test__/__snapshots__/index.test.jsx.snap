// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<First /> should match snapshot 1`] = `
<First>
  <div>
    <Second>
      <div
        className="second"
      >
        Second
      </div>
    </Second>
  </div>
</First>
`;
